Examples/ModalExample/ModalExample/AppDelegate.swift

```diff
@@ -25,7 +25,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         Iterate.shared.identify(userProperties: [
             "first_name": UserPropertyValue("Test"),
             "last_name": UserPropertyValue("User"),
-            "user_id": UserPropertyValue(123)
```

Comment:
on our test account in prod this property is a string not number, so switched to account_id so we can test with numbers